src/PhpSpec/Console/Command/DescribeCommand.php

```diff
@@ -64,9 +67,52 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $container = $this->getApplication()->getContainer();
         $container->configure();
 
-        $classname = $input->getArgument('class');
-        $resource  = $container->get('locator.resource_manager')->createResource($classname);
+        if ($input->getArgument('class')) {
+            $classname = $input->getArgument('class');
+        } else {
+            $questionHelper = $this->getApplication()->getHelperSet()->get('question');
+            $question = new Question('<info>Enter class to describe: </info>');
+
+            $question->setAutocompleterValues($this->getNamespaces());
+            $classname = $questionHelper->ask($input, $output, $question);
+        }
+
+        $resource = $container->get('locator.resource_manager')->createResource($classname);
 
         $container->get('code_generator')->generate($resource, 'specification');
     }
+
+    /**
+     * Get suites namespaces.
+     *
+     * @return array
+     */
+    private function getNamespaces()
+    {
+        $container = $this->getApplication()->getContainer();
+        $srcPaths = array();
+        $namespaces = array();
+
+        foreach ($container->getByTag('locator.locators') as $locator) {
+            if ($locator instanceof \PhpSpec\Locator\PSR0\PSR0Locator) {
+                $srcPaths[] = $locator->getSrcPath();
+            }
+        }
+
+        $finder = new Finder();
+        foreach ($finder->files()->name('*.php')->in($srcPaths) as $phpFile) {
+            preg_match('/namespace\s+(.*);/', $phpFile->getContents(), $matches);
```

Comment:
This would work for most cases but will be broken for this code:
```php
<?php
namespace Foo {
    class A {}
}
```

Also pattern can match text occurrences.
More reliable way of fetching namespace is using php tokenizer: http://php.net/manual/en/function.token-get-all.php

And BTW this would not work with projets that use PSR0 with underscores autoloading. I'm not sure if we should support it too for autoloading.